app/Lib/lang.php

```diff
@@ -704,6 +704,12 @@
                                  NameEnum::Official => 'Official',
                                  NameEnum::Preferred => 'Preferred'),
   
+  // Nested Group member filters
+  'en.nested.filters' => array(
+    'e'      => 'Exclude',
```

Comment:
A proper enum should be defined in `enum.php`.

Reply:
I've updated the values, but the issues remains. I'll add these to enum.php as well.

Reply:
Added - we now have NestedEnum of which both lang.php and the CoGroupMembersController make use.

Reply:
Update committed as of ee3fa84